Khurram_Mohammad-Abdullah_hw1

mohammadabdullahkhurram opened the pull request:
Homework 1 submission for Khurram, Mohammad Abdullah

kochaika reviewed (changes requested):
Your pull request deleting not your files!
This should be fixed!

mohammadabdullahkhurram commented:
> Your pull request deleting not your files! This should be fixed!

Dear Professor, 

Hope this finds you well. I removed the other folders that got attached to my repository when I copied it the first time. Those folders were not mine so I cleaned them up before submitting my homework. But all files of my homework have been submitted on time and with completion. Please let me know if there is anything else or anything I misunderstood. Thank you!
